website/public/js/controllers/notificationCtrl.js

```diff
@@ -58,35 +58,42 @@ habitrpg.controller('NotificationCtrl',
 
     $rootScope.$watch('user._tmp.drop', function(after, before){
       // won't work when getting the same item twice?
-      if (after == before || !after) return;
-      $rootScope.playSound('Achievement_Unlocked');
-      if (after.type !== 'gear') {
-        var type = (after.type == 'Food') ? 'food' :
-          (after.type == 'HatchingPotion') ? 'hatchingPotions' : // can we use camelcase and remove this line?
-          (after.type.toLowerCase() + 's');
+      if (_.isEqual(after, before) || !after) return;
+      var text, notes, type;
+      $rootScope.playSound('Item_Drop');
+
+      if (after.type !== 'gear' && after.type !== 'Quest') {
+        if (after.type === 'Food') {
+          type = 'food';
+        } else if (after.type === 'HatchingPotion') {
+          type = 'hatchingPotions';
+        } else type = after.type.toLowerCase() + 's';
         if(!User.user.items[type][after.key]){
           User.user.items[type][after.key] = 0;
         }
         User.user.items[type][after.key]++;
       }
 
-      if(after.type === 'HatchingPotion'){
```

Comment:
I think we can do some meta programming here to mitigate this awful if/else block, but that can be a refactor we do after this is merged in.